crates/phactory/src/system/gk.rs

```diff
@@ -1182,6 +1201,16 @@ mod tokenomic {
             (actual_payout, actual_treasury)
         }
 
+        pub fn update_v_recover(&mut self, now_ms: u64, block_number: chain::BlockNumber) {
+            self.v_deductible = fp!(0);
+            self.v_update_at = now_ms;
+            self.v_update_block = block_number;
+
+            // stats
+            self.last_payout = fp!(0);
+            self.last_payout_at_block = block_number;
```

Comment:
Looks like the function body is redundant with the end of `fn update_v_heartbeat`. Should it call this function in update_v_heartbeat?

Reply:
Looked at it again. The two functions have different update to the fields. So I think we can make it a common function in the future when there are more overlap.

Reply:
OK